src/mlmc/sampling_pool_pbs.py

```diff
@@ -0,0 +1,311 @@
+import os
+import shutil
+import subprocess
+import re
+import pickle
+import json
+import glob
+from level_simulation import LevelSimulation
+from sampling_pool import SamplingPool
+from pbs_process import PbsProcess
+
+
+class SamplingPoolPBS(SamplingPool):
+
+    OUTPUT_DIR = "output"
+    JOBS_DIR = "jobs"
+    LEVEL_SIM_CONFIG = "level_{}_simulation_config"  # Serialized level simulation
+    JOB = "{}_job.sh"  # Pbs process file
+
+    def __init__(self, work_dir, job_weight=200000, job_count=0):
```

Comment:
This constructor is called by the user, who has no idea how to choose job_weight. We should limit the job wall time, which is already part of the PBS configuration. Then the weight of individual samples can be given by n_ops, ie.e their time estimate. The system should be robust against killed jobs, so never mind if the actual job wall time will be a little larger then estimated.

Reply:
job_count can not be provided at the time of construction can be possibly set through the calls of have_permanent_sample.

Reply:
job_count is removed from constructor in child branch (MS_new_design_failed), so I would make these changes there